### DISCUSSION TITLE
Mastodon as enabling abuse

### DISCUSSION THREAD

**Discussion by SoniEx2** [mastodon/mastodon#23669] [General]
Mastodon provides tools for admins to directly control users' social graphs, namely blocked and limited instances.

These tools are very useful for admins, and we shouldn't take them away from admins.

But why do we not give users recourse?

While at it: Defederation is by definition a technological feature, not a social one. Providing recourse isn't "solving social problems with technological solutions". This is an issue of tech's own making.

**Comment by trwnh** [comment]
#3890, #11218

**Reply by SoniEx2** [reply]
Sure. We propose going further: give users a way to sidestep instance suspends, client-side, with the help of an anonymizing network (tor, i2p, etc).

**Reply by markcellus** [reply]
Note that #3890 and #11218 seem only related to [limited instances](https://docs.joinmastodon.org/admin/moderation/#limit-server) (previously called "silenced instances"). As I understand it, the ask here includes blocked instances as well... and maybe even suspended instances.

**Reply by SoniEx2** [reply]
Yeah. And let's have a honest discussion about the benefits, drawbacks, and technical tradeoffs of such a mechanism, while at it.

And while at it: no, we're not concerned about terrorists like chaya using this system. We have actual moderators across various instances, who can still ban users and limit discovery. Sure, this mechanism prevents yeeting users' connections/relationships, but you can still yeet users off your platform entirely. This mechanism would be purely harm reduction for marginalized communities which are actively being harmed by admins on the fedi today, on the threat of defederation. (Sorry for being so frustrated about this, but these are our communities too.)

**Reply by trwnh** [reply]
how would you sidestep a suspension? the point of a remote suspension, what it does, is to drop any incoming activity from the remote source. the only way to "sidestep" that is to be on a different server that does not do this. or, more to the point, to move away from the "server-as-a-community" model, make it a service provider only, and move the community to a higher layer (using things like groups and relays).

**Reply by SoniEx2** [reply]
yes please.

tho we were thinking more like discord: you have servers, you have users, and these are more or less independent. except servers provide discovery - without servers, no discovery.

can we move users to a higher layer? currently they're tightly coupled to servers.

(also yes, there should be a distinction between follow/following/mutual, and friend. the former is tied to an instance, the latter isn't.)

**Reply by trwnh** [reply]
it would be very difficult to completely separate users from servers because they are both bound to domain names.

**Reply by SoniEx2** [reply]
that can be treated as merely a discovery mechanism. it's used for follow/following/mutual, and for posting to an instance, but we can separate it from friends.

now, without syncing posts or anything, it wouldn't have much of a benefit, but it would make a good start. this could then be used for end-to-end encrypted DMs, if nothing else.

**Reply by edoswald** [reply]
I see this as more of a cultural issue than one that needs changing of the entire platform for. Too many admins use Defederation for inane BS rather than legitimate reasons. However, as an admin, I've defederated from certain servers for a reason, and it's always for widespread abuse, not certain accounts, or because I don't like the admins (journa.host and mastodon.social is a great example of this). 

I'd fully support adding to the server covenant some language about defederation, because I think it's used too often as revenge or petty crap. But forcing admins to carry content they object to (and in most cases pay for) isn't the way to go.

**Reply by SoniEx2** [reply]
sidestepping (de)federation using p2p (or other instances) is *not* forcing admins to carry content they object to (or is illegal or etc), it's the complete opposite.

putting language about defederation in the server covenant, on the other hand, *would* be pushing admins into hosting content they object to.
